README.md

```diff
@@ -2,30 +2,188 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Practice with the `Random` class.
+- Practice using the `if` statement.
+- Practice using chained conditionals.
+- Practice debugging `if` statements and observe the control flow.
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+In this lab, practice using the `Random` class and `if` statements to simulate
+a couple of games. We'll also practice debugging some more to observe the
+control flow when using `if` statements.
+
+Fork and clone this repository. When you do, you will see two files: `Lab.java`
+and `BuggyLab.java`. This assignment will have you write your own program to
+simulate some games along with debugging an existing program. Follow the
+instructions below to see where your code should go.
 
 ## Instructions
 
-Walk the student through any setup required to run the lesson (i.e.
-`npm install` and `npm start`).
+There are three parts to this lab:
+
+- Simulate flipping a coin.
+- Simulate rolling a die.
```

Comment:
I think this needs updated to rock paper scissors.

Reply:
Yes! Thanks for catching this!